lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py

```diff
@@ -89,6 +94,38 @@
             ports:
                 description:
                     - List of ports exposed within the container group.
+            environment_variables:
```

Comment:
is it an overwrite updating? how to remove an environment variable?

Reply:
yes, it's overwrite, doesn't work like tags